lib/widgets/drop_down.dart

```diff
@@ -1,42 +1,52 @@
 import 'package:dropdown_button2/dropdown_button2.dart';
 import 'package:flutter/material.dart';
 
-class DropDown extends StatefulWidget {
-  List<String> items;
+class DropDown extends StatelessWidget {
+  final String? dropdownValue;
+  final List<String> items;
+  final Function didSelectItemHandler;
 
-  DropDown(this.items, {super.key});
+  const DropDown(
+      {required this.items,
+      required this.didSelectItemHandler,
+      this.dropdownValue,
+      super.key});
```

Comment:
The layout of this constructor seems off, could you add a comma and make the IDE re-align?

```suggestion
      super.key,});
```

Reply:
done